carma_cooperative_perception/test/test_msg_conversion.cpp

```diff
@@ -118,7 +119,7 @@ TEST(ToDetectionMsg, Simple)
   EXPECT_DOUBLE_EQ(detection.accel.accel.linear.y, 1.0);
   EXPECT_NEAR(detection.accel.accel.linear.z, 2.4 * 9.80665, 1e-4);
 
-  EXPECT_EQ(detection.id, std::to_string(0xBEEF));
+  EXPECT_EQ(detection.id, "BADDCAFE-1");
```

Comment:
So this goes back to the conversation we had for https://github.com/usdot-fhwa-stol/carma-platform/pull/2279. Would it be possible by chance to have identical track names that are supposed to be representing different objects (I know we are making detections here, but I believe tracks just directly use the detection id when first created). 
Source1: {0xB1, 0xDD, 0xCA, 0xFE} first object -> `B1DDCAFE-1` -----> (when turned into external object, we only leave digits I believe, so) `11`?
Source2: {0xC1, 0xDD, 0xCA, 0xFE} first object -> `C1DDCAFE-1` ------> also `11`?

Reply:
Yes, that's a limitation on the track->external object conversion. We can open a new issue for that. I think we can implement a monotonically-increasing number generator so that all new tracks (regardless of their originating detection's ID) will have a unique value. Thoughts?

Reply:
Yeah I think that works too. Let's do that